src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef __WINDOWS__
+#include "resource.h"
+#elseif __LINUX__
```

Comment:
I think it is better to use `#elif` here. It is shorter, more common and consistent with the rest of the mkxp source code. (Also, I did not find documentation if `#elseif` is even supported under MSVC, but that is just an assumption)